src/daemon/daemon.h

```diff
@@ -173,6 +177,8 @@ public slots:
     std::unordered_set<std::string> allocated_mac_addrs;
     DaemonRpc daemon_rpc;
     QTimer source_images_maintenance_task;
+    multipass::utils::AsyncPeriodicTask<void> update_manifests_all_task{
+        "fetch manifest periodically", std::chrono::seconds(10), &Daemon::update_manifests_all, this, false};
```

Comment:
Yep, I kept it like this for testing reason since we are still iterating; however, maybe it is better to change it now and apply local change to test.

Reply:
Yeah, for this review, let's change it to how we want it to work, so I can review the actual behavior.  Thanks!